.github/workflows/test_upload_build_to_firebase.yml

```diff
@@ -0,0 +1,78 @@
+name: Test Upload Build to Firebase
+
+# SECRETS needed:
+### SSH_PRIVATE_KEY for Match Repo
+### MATCH_PASS
+### FIREBASE_TOKEN
+### STAGING_FIREBASE_APP_ID
+
+on:
+  pull_request
+  
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    name: Build
+    runs-on: macOS-latest
+    steps:
+    - name: Checkout Repo
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Install SSH key
+      uses: webfactory/ssh-agent@v0.5.4
```

Comment:
Can we use webfactory/ssh-agent@v0.7.0 ?

Reply:
@nkhanh44 was working on this in this https://github.com/nimblehq/ios-templates/pull/472